app/controllers/api/v2/os_default_templates_controller.rb

```diff
@@ -63,12 +65,19 @@ def destroy
       private
 
       def rename_config_template
-        if !params[:config_template_id].nil?
+        if params[:config_template_id].present?
           params[:provisioning_template_id] = params.delete(:config_template_id)
           Foreman::Deprecation.api_deprecation_warning("Config templates were renamed to provisioning templates")
         end
       end
 
+      def rename_nested_config_template
+        if params[:os_default_template] && params[:os_default_template][:config_template_id].present?
```

Comment:
I think we need to handle unwrapped parameters too, come to think of it - as they're meant to be the main way of using create/update.  Perhaps by defining `wrap_parameters` (see CR or Users controllers) and adding config_template_id so it gets copied into the wrapped hash?

61bdcf43 may have the same problem.